admin/database/database.go

```diff
@@ -115,9 +115,14 @@ type DB interface {
 	CheckUserIsAProjectMember(ctx context.Context, userID, projectID string) (bool, error)
 
 	InsertUsergroup(ctx context.Context, opts *InsertUsergroupOptions) (*Usergroup, error)
+	UpdateUsergroupName(ctx context.Context, name, groupID string) (*Usergroup, error)
+	DeleteUsergroup(ctx context.Context, groupID string) error
+	FindUsergroupByName(ctx context.Context, name, orgID string) (*Usergroup, error)
```

Comment:
Consider changing this to `FindUsergroupByName(ctx, orgName, name)`. This would be similar to `FindProjectByName(ctx, orgName, name)`. As far as I can tell, most places it is used, that would save a separate lookup for the org. The usergroup returned here will contain the org ID to use for permission checks.